### PR TITLE
updated tooltip and button

### PR DESCRIPTION
**Pull request by abrande**
Upgraded tooltip and button to helix-react. 
Button is more verbose and reusable -- removed Submit button for this.

### CODE REVIEW

File: app/components/SignUp/BillingInfo/BillingInfoForm.js [hunk 1]
```diff
@@ -73,9 +72,11 @@ export class BillingInfoForm extends React.Component {
                 />
               </div>
               <div className="hxCol hxSpan-6 align-right">
-                <Submit
-                  classNames="btn-wide hxBtn hxPrimary"
+                <Button
                   label={t('common:actions.basic.next')}
+                  variant="primary"
+                  size="wide"
```
Comment (nicko-winner) [inline app/components/SignUp/BillingInfo/BillingInfoForm.js:78]: do you mean to pass wide as a prop here?
```suggestion
                  size="wide"
                  wide
```

File: app/components/SignUp/CustomerInfo/CustomerInfoForm.js [hunk 2]
```diff
@@ -67,10 +68,11 @@ export class CustomerInfoForm extends React.Component {
           <div className="NavButtons">
             <div className="hxRow">
               <div className="hxCol hxSpan-12 align-right">
-                <Submit
-                  classNames="btn-wide hxBtn hxPrimary"
+                <Button
                   label={t('common:actions.basic.next')}
                   submit
+                  variant="primary"
+                  size="wide"
```
Comment (nicko-winner) [inline app/components/SignUp/CustomerInfo/CustomerInfoForm.js:75]: do you mean to pass wide as a prop here?

File: app/components/helix/buttons/Button.js [hunk 3]
```diff
@@ -1,20 +1,39 @@
...
+        className={buttonClassNames} // helix automatically adds a space between prefilled and added classes
         type={submit ? 'submit' : 'button'}
         disabled={disabled}
         onClick={onClick}
+        variant={variant}
+        size={size}
+        busy={processing}
       >
         {!processing && (label || children)}
         {processing && <span>{label || children}</span>}
```
Comment (nicko-winner) [inline app/components/helix/buttons/Button.js:35]: non blocking: ternary might be a little more clear
```suggestion
        {processing ? <span>{label || children}</span> : (label || children)}
```

File: app/components/SignUp/CustomerInfo/CustomerInfoForm.js (hunk 2 above)
Comment (abrande) [reply app/components/SignUp/CustomerInfo/CustomerInfoForm.js:75]: Yes but it should be a bool. Thanks

File: app/components/helix/buttons/Button.js [hunk 4]
```diff
@@ -1,20 +1,38 @@
...
+    } = this.props;
+    const buttonClassNames = classnames(classNames, {
+      'submit-btn': submit,
+      'btn-wide': wide
+    });
     return (
-      <button // eslint-disable-line react/button-has-type
-        className={`hxBtn ${classNames}`}
+      <HelixButton
+        className={buttonClassNames} // helix automatically adds a space between prefilled and added classes
```
Comment (nicko-winner) [inline app/components/helix/buttons/Button.js:26]: Thanks will fix the extra space in helix-react

**Review by nicko-winner — APPROVED** [review]
Code looks good. Please just make sure to test to make sure it works/looks as expected as we are early adopters of an Alpha version of helix react.

**Review by anuritjain — APPROVED** [review]

**Pull request merged**